modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -367,6 +406,10 @@ class IECallContext
     // Input parameters passed to an inference operation.
     cv::GArgs m_args;
     cv::GShapes m_in_shapes;
+
+    // keep alive preprocessed frames
+    std::mutex keep_alive_frames_mutex;
+    std::unordered_map<req_key_t, cv::MediaFrame> keep_alive_pp_frames;
```

Comment:
Suggest:
prepare_keep_frame_alive_slot
release_frame_kept_alive    _or at least_    release_keep_frame_alive

keep_frames_alive_mutex
(std::map) keep_pp_frames_alive

Reply:
OK, I've found a word "`retain`" which is "continue to have (something); keep possession of; hold fixed", so it's with the same meaning, but one-word

Or you may simply go with `hold`, also suits I think

so
prepare_retain_frame_slot (prepare "retain frame" slot)
release_retained_frame
retain_frames_mutex
(std::map) retain(ed)_pp_frames